server/src/test/java/org/elasticsearch/index/mapper/AbstractScriptFieldTypeTestCase.java

```diff
@@ -115,7 +116,7 @@ public void testMultiFieldsIsNotSupported() throws IOException {
             b.startObject("fields").startObject("test").field("type", "keyword").endObject().endObject();
         });
         MapperParsingException exception = expectThrows(MapperParsingException.class, () -> createMapperService(mapping));
-        assertEquals("Failed to parse mapping: runtime field [field] does not support [fields]", exception.getMessage());
+        assertThat(exception.getMessage(), containsString("unknown parameter [fields] on mapper"));
```

Comment:
if we do duplicate the code, I wonder if the message could be tailored to runtime fields, like it used to be. If we decide not to duplicate the code , I guess there is no way around it. In general, I am surprised that we call this "mapper": do users even know what a mapper is?

Reply:
I've replaced this error message to refer to `runtime field` instead of `mapper`